minidump/minidumpfile.py

```diff
@@ -183,10 +192,12 @@ def __parse_header(self):
 		self.header = MinidumpHeader.parse(self.file_handle)
 		for i in range(0, self.header.NumberOfStreams):
 			self.file_handle.seek(self.header.StreamDirectoryRva + i * 12, 0 )
-			self.directories.append(MINIDUMP_DIRECTORY.parse(self.file_handle))
-
+			minidump_dir = MINIDUMP_DIRECTORY.parse(self.file_handle)
+			if minidump_dir:
+				self.directories.append(minidump_dir)
 
```

Comment:
Please add a log line here that there is a stream that was ignored of type X

Reply:
but actually it's not needed

Reply:
ok done.